creation/create_cvmfsexec_distros.sh

```diff
@@ -156,19 +156,28 @@ error_handler() {
 
 ####################### MAIN SCRIPT STARTS FROM HERE #######################
 
-# parsing the command-line arguments
-# check whether the first argument passed is the option '--work-dir'
-if [[ $1 == "--work-dir" && ! -d "$2" ]]; then
-	# if value after this option is not a path (it is empty/unspecified), invoke the error handler
-	error_handler "The value of --work-dir option must be an existing directory."
+## parsing the command-line arguments
+# if only one argument and the argument value is -h or --help, print usage information
+if [[ $# -eq 1 && ( $1 == "-h" || $1 == "--help" ) ]]; then
```

Comment:
Probably could just check for `$1` w/o `$#`

Reply:
this has been addressed